eldritch/mythos.py

```diff
@@ -216,6 +216,98 @@ def get_interrupt(self, event, state):
     return None
 
 
+class Mythos7(Headline):
+  def __init__(self):
+    super().__init__("Mythos7", "Unnamable", "Woods", {"hex"}, {"slash", "triangle", "star"})
+    self.active_until = None
+
+  def create_event(self, state):
+    seq = super().create_event(state)
+    for char in state.characters:
+      if char.arrested_until is not None:
+        seq.events.append(events.ClearStatus(char, "arrested"))
+    seq.events.append(events.AddGlobalEffect(self, source_deck=state.mythos))
+    self.active_until = state.turn_number + 1
+    return seq
+
+  def get_interrupt(self, event, state):
+    if isinstance(event, events.Arrested):
+      return events.CancelEvent(event)
+    return None
+
+  def get_trigger(self, event, state):
+    if state.turn_number > self.active_until and isinstance(event, events.Turn):
```

Comment:
Technically it was remaining active until the end of the first player's Upkeep phase after the turn_number it was supposed to disappear (since `Upkeep` is a `Turn`). But that's needlessly confusing and introduces a small risk of something funky happening for other "until the end of the next turn" cards. I don't think there's any way to be arrested during upkeep.